src/rest/AnalyticsEndpoint.ts

```diff
@@ -93,22 +95,40 @@ export class AnalyticsEndpoint {
     return this.getFromService<string[]>(url, params);
   }
 
+  public forgetVisitorId() {
+    Cookie.erase('visitorId');
```

Comment:
Another cookie key we should clear is the `visitId`.

Reply:
Good point, I added another `Cookie.erase`.